bin/epaa.py

```diff
@@ -679,12 +679,25 @@ def get_matrix_max_score(allele, length):
         return np.nan
 
 
+def create_score_values(j, method):
+    if not pd.isnull(j):
+        if 'syf' in method:
+            return round(j, 4)
+        else:
+            # convert IC50 -> affinity score in [0,1]
+            min_ic50 = 1.0
+            return round(1.0 - math.log(max(j,min_ic50), 50000), 2)
+    else:
+        return np.nan
+
+
 def create_affinity_values(allele, length, j, method, max_scores, allele_strings):
     if not pd.isnull(j):
         if 'syf' in method:
             return round(((100.0 / float(max_scores[allele_strings[('%s_%s' % (str(allele), length))]]) * float(j)) / 100.0) * 100, 2)
         else:
```

Comment:
I think it would be even better if we test here if 'nuggets' or 'flurry' is used and keep the transformation from scores to affinities in case we want to use a tool in the future that returns scores.